content/news/2025-01-22_rocur-relaunch/index.en.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "We're Launching the R-Ladies Rotating Curation (RoCur) Account on Bluesky!"
```

Comment:
```suggestion
title: "We're Launching the R-Ladies Rotating Curation Account on Bluesky (RoCur)!"
```

I feel that where it is right now, "(RoCur)" breaks the flow.

Reply:
Removing abbreviation all together from the title; "RoCur" will be introduced in the first sentence of the main body instead